### DISCUSSION TITLE
Frigate LXC and go2rtc

### DISCUSSION THREAD

**Discussion by woody4165** [community-scripts/ProxmoxVE#482] [General]
Hi all

I've installed Frigate via the script and it works fine except for go2rtc that does not work.
I've followed the thread in the script description anf at the end it suggest to download the latest go2rtc version, actually 1.9.7, and I've saved it as executable in the /config folder. And it's working fine

The go2rtc version dowloaded with the script is 1.9.4 and won't work

Is it possible to update the script with the 1.9.7 version of go2rtc ?

Thanks

**Comment by clee604** [comment]
I did it successfully updated by replacing the file here in the LXC:
/usr/local/go2rtc/bin

My go2rtc worked fine though before the update though.

**Comment by fbloemhof** [comment]
I had a weird issue with go2rtc, it seems that Frigate was unable to create the streams for go2rtc.

I renamed the /usr/local/go2rtc folder and re-executed the following lines from the installation script, after a reboot all was working fine.

```
mkdir -p /usr/local/go2rtc/bin
cd /usr/local/go2rtc/bin
wget -qO go2rtc "https://github.com/AlexxIT/go2rtc/releases/latest/download/go2rtc_linux_amd64"
chmod +x go2rtc
$STD ln -svf /usr/local/go2rtc/bin/go2rtc /usr/local/bin/go2rtc
```
